content/product-engineering/engineering/cloud/devops/continuous_delivery.md

```diff
@@ -0,0 +1,13 @@
+# Sourcegraph Cloud Continuous Delivery
```

Comment:
```suggestion
# Sourcegraph Cloud continuous delivery
```

IIRC style guide says sentence case

Reply:
Also I think this is a duplicate of this page: https://handbook.sourcegraph.com/product-engineering/engineering/deployments